plugins/ui/src/deephaven/ui/object_types/ElementMessageStream.py

```diff
@@ -302,12 +305,29 @@ def _make_request(self, method: str, *params: Any) -> dict[str, Any]:
             "id": self._get_next_message_id(),
         }
 
-    def _send_document_update(self, root: RenderedNode) -> None:
+    def _make_dispatcher(self) -> Dispatcher:
+        dispatcher = Dispatcher()
+        dispatcher["setState"] = self._set_state
+        return dispatcher
+
+    def _set_state(self, state: dict[str, Any]) -> None:
+        """
+        Set the state of the element. This is called by the client on initial load.
+
+        Args:
+            state: The state to set
+        """
+        logger.debug("Setting state: %s", state)
+        self._context.import_state(state)
+        self._mark_dirty()
+
+    def _send_document_update(self, root: RenderedNode, context: RenderContext) -> None:
         """
         Send a document update to the client. Currently just sends the entire document for each update.
 
         Args:
             root: The root node of the document to send
+            state: The state of the node to preserve
```

Comment:
not sure if this or rename the variable is better
```suggestion
            context: The state of the node to preserve
```

Reply:
Changed it to just take the exported state instead.